Added Partition Key Parameter for Kafka Producer (#8335)
RasaHQ/rasa: docs/docs/event-brokers.mdx

events. Rasa uses the [python-kafka](https://kafka-python.readthedocs.io/en/master/usage.html) library, a Kafka client written in Python.
You will need a running Kafka server.

### Partition Key

> Rasa's Kafka producer can optionally be configured to partition messages by sender_id.

Reviewer: ```suggestion
Rasa Open Source's Kafka producer can optionally be configured to partition messages by conversation ID.
```

> This can be configured by setting `partition_by_sender` in the endpoints.yml file to True.

Reviewer: ```suggestion
This can be configured by setting `partition_by_sender` in the `endpoints.yml` file to True.
```

> By default, this parameter is set to False and the producer will randomly assign a partition to each message.  

Reviewer: ```suggestion
By default, this parameter is set to `False` and the producer will randomly assign a partition to each message.  
```
